projects/ccd-case-ui-toolkit/src/lib/shared/components/search-result/search-result.component.html

```diff
@@ -33,20 +33,16 @@ <h2 class="heading-h2" id="search-result-heading__text" tabindex="-1">{{ (caseSt
           </label>
         </div>
       </th>
-      <th *ngFor="let col of resultView.columns">
-        <table class="search-result-column-header"
-          [attr.aria-label]="('Sort by ' + col.label + ' ' + isSortAscending(col)? 'ascending' : 'descending') | rpxTranslate">
-          <tr>
-            <th>
-              <div class="search-result-column-label" (click)="sort(col)">
-                {{col.label | rpxTranslate}}
-              </div>
-              <div *ngIf="comparator(col)" class="search-result-column-sort">
-                <a (click)="sort(col)" class="sort-widget" [innerHTML]="sortWidget(col)" href="javascript:void(0)"></a>
-              </div>
-            </th>
-          </tr>
-        </table>
+      <th *ngFor="let col of resultView.columns" class="search-result-column-header"
+        [attr.aria-sort]="isSortAscending(col) === null ? null : (isSortAscending(col) ? 'ascending' : 'descending')">
+        <div>
+          <div class="search-result-column-label" (click)="sort(col)" (keyup)="noop()">
```

Comment:
What is the purpose of '(keyup)="noop()"' 
At present if you tab to the sort header on the results and hit enter you can change the sort order. Can you confirm that this still works?
<img width="549" alt="image" src="https://github.com/hmcts/ccd-case-ui-toolkit/assets/49269487/dd01fa22-2660-4dd9-9cf6-204e2e328f47">

Reply:
@andywilkinshmcts Have confirmed that the functionality still works, needed to add the (keyup) from sonarcloud reliability/maintainability